scripts/migrate-audit-enabled-default.sh

```
@@ -0,0 +1,74 @@
+
+if [ -n "${1:-}" ]; then
+    # single instance mode
+    path="$baseDirectory/${PREFIX:-}$1"
+    if [ ! -d "$path" ]; then
+        echo "Instance directory not found: $path"
+        exit 1
+    fi
+    migrate_instance "$path"
+else
+    # all instances
+    cd "$baseDirectory"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)